runtime/doc/vim_diff.txt

```diff
@@ -704,6 +704,7 @@ Options:
 - *'ballooneval'* *'beval'* *'noballooneval'* *'nobeval'*
 - *'balloonexpr'* *'bexpr'*
 - bioskey (MS-DOS)
+- *'browsedir'* *'bsdir'*
```

Comment:
this is probably something we want to restore, should `:help nvim-missing` rather than `nvim-removed`